_data/gambling.yml

```diff
@@ -24,7 +24,7 @@ websites:
       othercrypto: Yes
 
     - name: Bitcoin Games
-      url: https://games.bitcoin.com
+      url: https://games.bitcoin.com/videopoker
```

Comment:
Why change linking to the main landing page?

Reply:
because the first one redirects to the other, and that makes Travis annoyed so I just went direct

Reply:
Makes sense!